pyproject.toml

```diff
@@ -113,6 +113,7 @@ ignore = [
 exclude_dirs = ["tests"]
 
 [tool.pytest.ini_options]
+pythonpath = [".", "src"]
```

Comment:
Where does this come from? Should not be necessary or?

Reply:
In my repo I was getting an Error with pytest without it. 

```bash
ImportError while loading conftest '/Path/to/conftest.py'.
tests/conftest.py:16: in <module>
    from lakefs_spec import LakeFSFileSystem
E   ModuleNotFoundError: No module named 'lakefs_spec'
```

Reply:
I thought it is nicer to configure the path in the `.toml` rather than setting it up manuall w.r.t. possible future contributors.

Reply:
How do you invoke `pytest`? From the contributing guide, everything works fine without this change on my machine - I'm therefore inclined to troubleshoot the issue on your PC and leave it as is.

Reply:
I just do `pytest` in the root directory. But sure, I'll leave as is.